lib/engine/game/g_18_co.rb

```diff
@@ -116,6 +119,10 @@ def sell_shares_and_change_price(bundle)
         log_share_price(corporation, price) if self.class::SELL_MOVEMENT != :none
       end
 
+      def legal_tile_rotation?(_entity, _hex, tile)
+        super unless TILES_FIXED_ROTATION.include?(tile.name) && tile.rotation != 0
```

Comment:
don't use unless with multiple statements

Reply:
I've switched back to the previous version using if rather than unless